test/NuGet.Clients.FuncTests/NuGet.CommandLine.FuncTest/Commands/RestoreCommandTests.cs

```diff
@@ -24,6 +24,7 @@ public class RestoreCommandTests
     {
         private const int _successExitCode = 0;
         private const int _failureExitCode = 1;
+        private string _httpError = "You are running the 'restore' operation with an 'HTTP' source: {0}. NuGet requires HTTPS sources. To use an HTTP source, you must explicitly set 'allowInsecureConnections' to true in your NuGet.Config file. Please refer to https://aka.ms/nuget-https-everywhere.";
```

Comment:
Similar as another PR. 
Can we try not to use a hardcoded message and use a resource if possible?